CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [2.2.0] - 2017-09-05
```

Comment:
This version is appropriate, I'll update the version when this is merged.
Usually I put the date as `unreleased` until I publish. I can fix this though when I do a release.